docs/components/gantry/_index.md

```diff
@@ -44,7 +44,7 @@ Most robots with a gantry need at least the following hardware:
 
 ## Supported Models
 
-To use your gantry with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your gantry.
+To use your gantry with Viam, check whether one of the following [built-in models](#built-in-models) or {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} supports your gantry.
```

Comment:
in this case the link is supposed to go to lower on this page, so I think it makes sense to keep
```suggestion
To use your gantry with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your gantry.
```